### PR TITLE
chore(blog): Blog post about glob matching

### PR DESCRIPTION
**Pull request by hermanschaaf**
A short blog post about glob matching.

The plugins are still being released, but when I have the version numbers for them I will update the final part of the post.

### CODE REVIEW

File: website/pages/blog/introducing-glob-matching-for-tables.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+ - AWS [v4.15.0](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-aws-v4.15.0)
+ - GCP [v2.4.15](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-gcp-v2.4.15)
+ - Azure: [v1.4.4](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-azure-v1.4.4)
+ 
+See [this link](https://www.cloudquery.io/docs/plugins/sources) for a list of the latest versions of all official source plugins.
+
+## Conclusion
+
+This is a small change, but sometimes small changes make a big difference. If you have any thoughts or feedback about the feature, please share them as a [Github issue](https://github.com/cloudquery/cloudquery) or chat to us on [Discord](https://www.cloudquery.io/discord)! ❤️
```
Comment (erezrokah) [inline website/pages/blog/introducing-glob-matching-for-tables.md:95]:
```suggestion
This is a small change, but sometimes small changes make a big difference. If you have any thoughts or feedback about the feature, please share them as a [GitHub issue](https://github.com/cloudquery/cloudquery) or chat to us on [Discord](https://www.cloudquery.io/discord)! ❤️
```

Comment (erezrokah) [inline website/pages/blog/introducing-glob-matching-for-tables.md:95]: Maybe rephrase this? I get that the implementation might have be a small change, but I'm worried about miscommunication the impact.
Also it was big enough to write a blog post about it.

Maybe:
```
With Glob matching support, we're hoping to reduce a lot of the boilerplate required to configure tables to sync with CloudQuery. We'd be happy to learn of any thoughts or feedback...
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
Great feature! Added a few comments

**Review by erezrokah — APPROVED** [review]

File: website/pages/blog/introducing-wildcard-matching-for-tables.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+ - AWS [v4.15.0](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-aws-v4.15.0)
+ - GCP [v2.4.15](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-gcp-v2.4.15)
+ - Azure: [v1.4.4](https://github.com/cloudquery/cloudquery/releases/tag/plugins-source-azure-v1.4.4)
+ 
+See [this link](/docs/plugins/sources) for a list of the latest versions of all official source plugins.
+
+## Conclusion
+
+With wildcard matching support, we're hoping to reduce a lot of the boilerplate required to configure tables to sync with CloudQuery. If you have any thoughts or feedback about the feature, please share them as a [GitHub issue](https://github.com/cloudquery/cloudquery) or chat to us on [Discord](/discord)! ❤️
```
Comment (erezrokah) [inline website/pages/blog/introducing-wildcard-matching-for-tables.md:95]:
```suggestion
With wildcard matching support, we're hoping to reduce a lot of the boilerplate required to configure tables to sync with CloudQuery. If you have any thoughts or feedback about the feature, please share them as a [GitHub issue](https://github.com/cloudquery/cloudquery) or chat to us on [Discord](https://cloudquery.io/discord)! ❤️
```

My back, looks like we exclude this in our broken links check as it's a redirect, so using the Deploy Preview doesn't work and fails the broken links checker. Let's keep this one on the main domain

**Review by yevgenypats — APPROVED** [review]
Looks good! Let's also put it in the docs/recipes!
